pac4j-core/src/main/java/org/pac4j/core/profile/definition/CommonProfileDefinition.java

```diff
@@ -33,6 +34,7 @@ public CommonProfileDefinition() {
         primary(PICTURE_URL, Converters.URL);
         primary(PROFILE_URL, Converters.URL);
         primary(LOCATION, Converters.STRING);
+        primary(USERNAME, Converters.STRING);
```

Comment:
You can use the `Pac4jConstants.USERNAME` instead of creating a new one...

Reply:
Fixed